lib/js/stanford_person.behavior.js

```diff
@@ -21,6 +21,12 @@ export default {
           $(this).attr('aria-expanded', 'false');
         }
       });
+
+      $('.stanford-people-grid--filters h3').each(function(index) {
+        var label = $(this).text();
+        $(this).addClass(index + ':' + label);
```

Comment:
This approach seems fine, but we should change the semicolon and adjust the label before setting the class. Also I don't believe starting with a number is good practice. Perhaps switching them around so the result is `student--index-0`.
Think about the scenario `faculty` and `faculty emeritus` that would result in `<h3 class="0:faculty emeritus">` which would be treated the same as `0:faculty`

We should be able to accomplish that with `label.replace(/[\W_]+/g,"-");`

Reply:
> label.replace(/[\W_]+/g,"-");

Whenever you see a problem and think, "I know, I'll solve this with regular expressions!"

Well, now you have two problems.

Reply:
How about this way instead of using classes?